released/packages/coq-geocoq/coq-geocoq.2.3.0/opam

```diff
@@ -17,7 +17,7 @@ install: [
 ]
 remove: ["rm" "-R" "%{lib}%/coq/user-contrib/GeoCoq"]
 depends: [
-  "coq" { (>= "8.5" & < "8.6~") | (>= "8.6.1" & < "8.9~") }
+  "coq" { (>= "8.5" & < "8.6~") | (>= "8.6.1" & <= "8.9~") }
```

Comment:
You should use `< "8.10~"` since `X~` is the smallest number grater than `X`, it won't include `8.9.1` for example.